src/index.html

```diff
@@ -1,13 +1,431 @@
+
+                <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-4">
+                  <div class="place__photo-container place__photo-container--tablet">
+                    <img
+                    src="./images/photos/3.svg"
+                    alt="photo3"
+                    class="place__photo">
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--tablet-4-6 grid__item--desktop-5-8">
+                  <div class="place__photo-container place__photo-container--tablet">
+                    <img
+                      src="./images/photos/4.svg"
+                      alt="photo4"
+                      class="place__photo">
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--least grid__item--tablet-5-6 grid__item--desktop-9-12 ">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/5.svg"
+                      alt="photo5"
+                      class="place__photo">
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <p class="place__description--main">Bodhisattvas of Wisdom, Compassion, and Power</p>
+              <p class="place__description--text">March 27, 2021 – October 30, 2022</p>
+            </div>
+          </div>
+
+          <div class="place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--tablet-1-2 grid__item--desktop-1-4 ">
+                  <div class="place__photo-container">
+                    <img
+                    src="./images/photos/6.svg"
+                    alt="photo6"
+                    class="place__photo">
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--tablet-3-6 grid__item--desktop-5-12 ">
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/photos/7.svg"
+                      alt="photo7"
+                      class="place__photo place__photo--special">
+                  </div>
+                </div>
+
+              </div>
+            </div>
+            <div class="place__description">
+              <p
+                class="place__description--main"
+                >Inspiring Walt Disney: The Animation of French Decorative Arts
+              </p>
+              <p
+                class="place__description--text"
+                >December 10, 2021 – March 6, 2022
+              </p>
+            </div>
+          </div>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--tablet-2-5 grid__item--desktop-5-8">
+              <a
+                href="#"
+                class="button"
+                target="_blank">View all exhibitions</a>
```

Comment:
can't push it. show you the reason 
![image](https://user-images.githubusercontent.com/105124305/188450631-db13b2cf-df10-41fe-91cb-946eda1d9ce9.png)